config/src/config/network_config.rs

```diff
@@ -114,6 +115,7 @@ impl NetworkConfig {
             listen_address: "/ip4/0.0.0.0/tcp/6180".parse().unwrap(),
             mutual_authentication: false,
             network_id,
+            runtime_threads: 8,
```

Comment:
nit: I'd consider making this optional and only call `worker_threads` if it's specified. Reason being is that this might slow things down on a large machine with more than 8 cores, and it might overload smaller machines? (e.g., those with 2 cores)? Have we tried it on a small machine? e.g., a PFN

Reply:
Sure @JoshLind - We can make this configuration as optional. Since this PR is merged, I will address this in a follow up PR.